R/utils.R

```diff
@@ -179,3 +179,9 @@ file_size <- function(...) {
 prompt_ask_yes_no <- function(reason) {
   utils::menu(c("no", "yes"), FALSE, title = reason) == 2 # nocov
 }
+
+#' @useDynLib storr
```

Comment:
this should be `@useDynLib storr, .registration = TRUE` in order for registration to work

Reply:
See 4f06d03782e2d7bbe2ee7f35b97e0fcb893dac86.